### DISCUSSION TITLE
createContractEventFilter address property

### DISCUSSION THREAD

**Discussion by dev-aly3n** [wevm/viem#1070] [Question]
Hi, I'm getting nothing when I set address property in `publicClient.createContractEventFilter` . (however I get events without this property)
I'm using contract address
is it a bug?

**Comment by ECBSJ** [comment]
I've also been having issues with `publicClient.createContractEventFilter` and `contractInstance.createEventFilter`. None of them are working for me.

Can I see a code snippet of what you have for this?

**Reply by dev-aly3n** [reply]
Sure, check it here:
https://github.com/Ithil-protocol/frontend/blob/mainnet/src/hooks/useClosePositions.ts#L43
